src/main.rs

```diff
@@ -47,11 +47,31 @@ fn shebang(nightly: bool, quiet: bool) -> String {
     format!("#!/usr/bin/env {cargo_invocation}{quiet_arg}")
 }
 
-fn frontmatter() -> &'static str {
-    "---
-[dependencies]
+fn frontmatter(release_profile: bool) -> String {
+    let mut buf = String::new();
+    
+    buf.push_str("---\n");
+    buf.push_str("[dependencies]\n\n");
 
----"
+    if release_profile {
+        buf.push_str("[profile.dev]\n");
+        buf.push_str(release_profile_settings());
+    }
+
+    buf.push_str("---");
+
+    buf
+}
+
+fn release_profile_settings() -> &'static str {
+    r#"opt-level = 3
+debug = false
+debuginfo = "None"
```

Comment:
I was a bit to quikc with merging this. When running a script without `--quiet` cargo prints a warning
```
warning: /home/alexander/Documents/dev/cargo-new-script/Cargo.toml: unused manifest key: profile.dev.debuginfo
```
I also don't see it mentioned in the pages you linked.

Reply:
whoops, you're right! thanks for taking care of this.